### PR TITLE
Entry point made mandatory

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 687899c4a2ed74258c0096d58c99393d6fb779a8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/services/catalogs/catalog_item.py [hunk 1]
```diff
@@ -3,14 +3,17 @@
...
 from cfme.web_ui import toolbar as tb
 from utils.update import Updateable
 from utils.pretty import Pretty
+from utils.version import current_version
 
 cfg_btn = partial(tb.select, "Configuration")
 accordion_tree = partial(accordion.tree, "Catalog Items")
 policy_btn = partial(tb.select, "Policy")
+dynamic_tree = Tree("//div[@id='basic_info_div']"
+                    "/div/div/div/div/div/div/ul[@class='dynatree-container']")
```
Comment (seandst) [inline cfme/services/catalogs/catalog_item.py]: Can we get a cleaner locator here? Perhaps `"//div[@id='basic_info_div']//ul[@class='dynatree-container']"`?

**Comment by seandst** [comment]
Please add some pytest options so that the affected tests are run through the pr tester.

**Comment by dajoRH** [comment]
Lint report for commit 8150318929ba0c2fb9dc601dd0f59c6405cb281c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**